cmd/build.go

```diff
@@ -238,6 +239,7 @@ against OPA v0.22.0:
 	buildCommand.Flags().VarP(&buildParams.revision, "revision", "r", "set output bundle revision")
 	buildCommand.Flags().StringVarP(&buildParams.outputFile, "output", "o", "bundle.tar.gz", "set the output filename")
 	buildCommand.Flags().StringVar(&buildParams.ns, "partial-namespace", "partial", "set the namespace to use for partially evaluated files in an optimized bundle")
+	buildCommand.Flags().BoolVar(&buildParams.followSymlinks, "follow-symlinks", false, "follow symlinks in the input directory when building the bundle")
```

Comment:
Are we limited to only following symlinks in directories, and not also direct paths to symlinks? Doesn't look like that'd be the case 🤔; i.e. the `FileLoader` created and called in `LoadPathsForRegoVersion` looks like it'd follow links.
If this isn't a limitation, we should update this description to say that symlinks in the provided set of paths are followed (not just dirs). And we should update the test to assert this (or add an additional test).

Reply:
Nope, we can follow direct paths to symlinks! Added a test accordingly